appointment/upload-documents/uploadDocuments.php

```diff
@@ -167,7 +168,10 @@ class="dcf-btn dcf-btn-primary dcf-mt-1"
 						<ul>
 							<b class="dcf-uppercase">Important Note</b>: If you are typing your information into the fillable form, you must "save as" to your device before uploading it or your information will not save.
 							<li><b>Completed <a href ng-click="downloadForm14446()">Form 14446</a></b>. The <b>bottom portion of page two</b> (Part III: Taxpayer Consents) needs to be completed.</li>
-							<li><b>Completed <a href ng-click="downloadResidentIntakeForm()">Form 13614-C</a></b> (Intake Form). Without this form, Lincoln VITA cannot prepare your return.</li>
+							<li><b>Completed <a href ng-click="downloadResidentIntakeForm()">Form 13614-C</a></b> (Intake Form for residents). Without this form, Lincoln VITA cannot prepare your return.</li>
+							<ul>
+								<li>The Spanish version of the Intake Form for residents can be found here: <b><a href ng-click="downloadResidentIntakeFormSpanish()">Form 13614-C (SP)</a></b></li>
```

Comment:
It feels odd to me to add another bullet just to present a version of the document in Spanish. I wonder if something like this might sound better? "Completed Form 13614-C (Intake form for residents). This is available in both an <link> English version</link> and a <link>Spanish version</link>. Without this form, Lincoln VITA cannot prepare your return."

Not sure what the exact verbiage would be--probably should be workshopped with Tracy/Dr. Moody, but this is my gut reaction